nc/source/time/NcTime.cpp

```diff
@@ -13,38 +13,21 @@ namespace nc::time
     using std::chrono::nanoseconds;
 
     Time::Time() noexcept
+        : m_lastTime{Clock::now()},
```

Comment:
We should eventually switch over and normalize using UTC time, especially when we get into networking.